doc/source/servers/batch.md

```diff
@@ -140,6 +140,53 @@ However currently the implementation uses the Seldon Client which does not lever
 
 ### Micro batching
 
-When using the batch processor CLI you can specify a `batch-size` parameter which can group multiple predictions into a single request. This allows you to take advantage of the higher performance this provides for some models, and reduce networking overhead. The response will be split back into multiple single prediction responses so that the output file looks identical to running the processor with a batch size of 1.
+When using the batch processor CLI you can specify a `--batch-size` parameter which can group multiple predictions into a single request. This allows you to take advantage of the higher performance this provides for some models, and reduce networking overhead. The response will be split back into multiple single prediction responses so that the output file looks identical to running the processor with a batch size of 1.
 
-Currently we only support micro batching for `ndarray` and `tensor` payload types.
+Currently we only support micro batching for `ndarray` and `tensor` payload types, as well as for `raw` data type of inputs.
```

Comment:
may be worth adding a link to "micro batching" concept that explains what it is (if relevant)

Reply:
not sure what link you have in mind?

Reply:
I can't really find a resourcee that isn't a personal blog that explains this so probably we can just leave it